packages/valory/skills/decision_maker_abci/policy.py

```diff
@@ -131,9 +131,8 @@ def best_tool(self) -> str:
     def update_weighted_accuracy(self) -> None:
         """Update the weighted accuracy for each tool."""
         self.weighted_accuracy = {
-            tool: (acc_info.accuracy / 100)
-            * (acc_info.requests - acc_info.pending)
-            / self.n_requests
+            tool: acc_info.accuracy
```

Comment:
Removed division by `100` so that the weighted accuracy is on the same scale as the accuracy, i.e., `[0 - 100]`, not `[0 - 1]`.

Reply:
the weighted accuracy cannot be between 0 and 100 because we are adding an extra number to the tool accuracy, which is the volume of requests factor

Reply:
since we are dividing now the volume factor by 2 this means the maximum number this variable can reach is 0.5 so the maximum number the weighted accuracy could reach is 100.5

Reply:
81c547d7d202568758c2fa2cc04a63b24b0e7bf1